core/plugins/org.polarsys.capella.core.validation.ui/src/org/polarsys/capella/core/validation/ui/DynamicActionProvider.java

```diff
@@ -105,6 +105,8 @@ public ValidateAction createDefaultValidation() {
     ISelection selection = selectionProvider.getSelection();
     if (!selection.isEmpty()) {
       validationAction.selectionChanged(new SelectionChangedEvent(selectionProvider, selection));
+    } else {
+      validationAction.setEnabled(false);
     }

```

Comment:
This change is for the case when an element was previously selected, but you click on empty space in SB (selection inside the validate action is empty).

![image](https://user-images.githubusercontent.com/60220371/113274814-c75fa680-92e6-11eb-85cf-d21a77eb6aae.png)